templates/managed-s3.yaml

```diff
@@ -0,0 +1,211 @@
+Description: Provision a generic or a Synapse bucket
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  AllowWriteBucket:
+    Type: String
+    Description: true for read-write, false (default) for read-only bucket
+    AllowedValues:
+      - true
+      - false
+    Default: false
+  SynapseBucket:
+    Type: String
+    Description: true for a synapse bucket, false (default) for general bucket
+    AllowedValues:
+      - true
+      - false
+    Default: false
+  EncryptBucket:
+    Type: String
+    Description: true (default) to encrypt bucket, false for no encryption
```

Comment:
Should this always just be true. No false allowed?

Reply:
well, on scicomp should be true always but on sandbox could go either way?